package-lock.json

```diff
@@ -1,222 +1,279 @@
 {
   "name": "hot-shots",
   "version": "10.1.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
And now obvious why the tests are failing, still supporting very old Node versions here.  I'll fix up this PR later for this.